src/main/java/frc/robot/RobotContainer.java

```diff
@@ -80,8 +81,8 @@ private void configureBindings() {
 
     drive.setDefaultCommand(
         new DefaultDrive(drive,
-            () -> getJoystickInput(driverController, Constants.Controller.LEFT_Y_AXIS),
-            () -> getJoystickInput(driverController, Constants.Controller.LEFT_X_AXIS),
+            () -> AllianceUtil.getTranslationDirection() * getJoystickInput(driverController, Constants.Controller.LEFT_Y_AXIS),
+            () -> AllianceUtil.getTranslationDirection() * getJoystickInput(driverController, Constants.Controller.LEFT_X_AXIS),
```

Comment:
Does the PS4 controller invert both the X and Y acis?

Reply:
Yes